Tasks/VsTest/vstest.ts

```diff
@@ -187,6 +187,28 @@ function generateResponseFile(): Q.Promise<string> {
     return defer.promise;
 }
 
+function publishCodeChanges(): Q.Promise<string> {
```

Comment:
i thnk we shuld not publish code changes if already done. shuld it be in tool or at task layer? what do u think. Also how will we do that?